openslides_backend/action/actions/user/participant_json_upload.py

```diff
@@ -105,8 +105,11 @@ def validate_entry(self, entry: dict[str, Any]) -> dict[str, Any]:
         entry.pop("structure_level_ids")
         entry.pop("meeting_id")
 
-        if "username" in failing_fields and not entry["username"].get("id"):
-            failing_fields.remove("username")
+        if not entry.get("id"):
```

Comment:
Why do you use the entry[id] instead of the user_name or member_number related one?

Reply:
Because it is the shortest way to check if this is a row update. If I didn't do it this way, I'd have to check for `not (entry["username"].get("id") or entry["member_number"].get("id")